docs/examples.md

```diff
@@ -360,6 +360,48 @@ lint_task:
   script: flake8 *.py
 ```
 
+### `Unittest` Annotations
+
+Cirrus CI annotations are available for Python Unittest results. This way you can see what tests are failing without leaving the pull request you are reviewing!
+
+This can be set up by doing the following:
+
+First, add tests if you haven't already. Here is a basic one that just makes sure you are running it from Cirrus CI:
```

Comment:
Not sure if people already reading about Unittest don't have tests already. 🤔

Reply:
True.